packages/playground/src/App.vue

```diff
@@ -336,16 +336,19 @@ async function setTimeouts() {
   }
 
   const client = gridStore.client as GridClient;
-  const clientOptions = client.clientOptions;
+
   const localStorageDeploymentTimeout = localStorage.getItem(LocalStorageSettingsKey.TIMEOUT_DEPLOYMENT_KEY);
-  const deploymentTimeoutMinutes = clientOptions.deploymentTimeoutMinutes;
-
-  if (client && clientOptions && deploymentTimeoutMinutes) {
-    if (!localStorageDeploymentTimeout) {
-      localStorage.setItem(LocalStorageSettingsKey.TIMEOUT_DEPLOYMENT_KEY, `${+deploymentTimeoutMinutes * 60}`);
-    } else {
-      client.clientOptions.deploymentTimeoutMinutes = +localStorageDeploymentTimeout! / 60;
-      await client.connect();
+
+  if (client) {
+    const clientOptions = client.clientOptions;
+    if (clientOptions) {
+      const deploymentTimeoutMinutes = clientOptions.deploymentTimeoutMinutes;
+      if (!localStorageDeploymentTimeout && deploymentTimeoutMinutes) {
+        localStorage.setItem(LocalStorageSettingsKey.TIMEOUT_DEPLOYMENT_KEY, `${+deploymentTimeoutMinutes * 60}`);
+      } else {
+        client.clientOptions.deploymentTimeoutMinutes = +localStorageDeploymentTimeout! / 60;
```

Comment:
```suggestion
        clientOptions.deploymentTimeoutMinutes = +localStorageDeploymentTimeout! / 60;
```
you already have a clientOptions variable that you can use directly

Reply:
doesn't update the timeout on the client. Note how I was still able to deploy here even though timeout is 3 seconds. 
![Screenshot from 2024-09-21 21-34-16](https://github.com/user-attachments/assets/87b40950-a3b2-4ad7-8844-9977017fa32b)
4d66ef1242